conf/messages.en

```diff
@@ -442,4 +446,25 @@ date.month.8 = August
 date.month.9 = September
 date.month.10 = October
 date.month.11 = November
-date.month.12 = December
\ No newline at end of file
+date.month.12 = December
+
+charityRegistrationNumber.title = What is your charity registration number (CHRN)?
+charityRegistrationNumber.heading = What is your charity registration number (CHRN)?
+charityRegistrationNumber.p1 = Your charity registration number is on any mail that the charity regulator has sent you.
+charityRegistrationNumber.p2 = In Scotland, your registration number will start with SC.
+charityRegistrationNumber.hint = For example, 01234567 or SC012345.
+charityRegistrationNumber.error.required = Enter a charity registration number
+charityRegistrationNumber.error.length = Charity registration number must not be more than {0} characters
+
+businessName.title = What is the name of your business?
+businessName.heading = What is the name of your business?
+businessName.hint = This is the registered name of your business.
+businessName.error.required = Enter a business name
+BusinessName.error.length = Business name must not be more than {0} characters
+
+companyRegistrationNumber.title = What is your company registration number?
```

Comment:
Should be :

What is your company registration number? - register for the Economic Crime Levy - GOV.UK

Reply:
@nemanjatomanovic you don't need the service name and GOV.UK in the message as this is automatically added by the `title` method in `ViewUtils` that we use

Reply:
Ok, I will resolve this thread. Thank you.